### DISCUSSION TITLE
Curious content assist in comments

### DISCUSSION THREAD

**Discussion by SJLennon** [codefori/vscode-ibmi#1802] [Q&A]
Is this normal? Should I be getting automatic content assist in comments? Is there a setting I'm missing?

I'm entering a comment at the end of a line and as soon as I hit space after the second / I get this:
![image](https://github.com/codefori/vscode-ibmi/assets/67484051/478419c4-b5fd-49a7-a1f2-6c34dd936c44)

**Comment by worksofliam** [comment]
Yes, this is normal. We allow the content assist to work in comment areas, and space is a trigger character.

**Reply by SJLennon** [reply]
Ummm.  It's a bit disconcerting in comments, because it pops up every time you hit the space bar between words. If I had my druthers, in comments I think I'd rather manually call for content assist with Ctrl+space.  But there are probably other more important things that need done.
